charts/flipt/templates/service.yaml

```
@@ -1,10 +1,36 @@
+{{- if (or (empty .Values.service.enabled) .Values.service.enabled))}}
```

```suggestion
{{- if (or (empty .Values.service.enabled) .Values.service.enabled)}}
```
Was a mistake in the github suggestion I made. Apologies.

---

@tomaszmichalowski just a heads up. One this fix is applied I think it is ready to merge (given tests pass).
I left it here for you to do the honours. Just to sanity check me.